tree/30_generic_methods/service_check_started_at_boot.cf

```diff
@@ -29,13 +29,13 @@ bundle agent service_check_started_at_boot(service_name)
 {
   vars:
 
-    systemctl_utility_present::
+    pass2.systemctl_utility_present.!broken_systemctl::
       "command_to_check"   string => "${paths.path[systemctl]} is-enabled ${service_name}";
 
-    !systemctl_utility_present.chkconfig_utility_present::
+    pass2.!systemctl_utility_present.chkconfig_utility_present::
       "command_to_check"   string => "${paths.path[chkconfig]} --list ${service_name} | grep -q -e 3:on -e B:on";
 
-    !systemctl_utility_present.!chkconfig_utility_present::
+    pass2.!(systemctl_utility_present|chkconfig_utility_present)|broken_systemctl::
```

Comment:
you mean
pass2.(!(systemctl_utility_present|chkconfig_utility_present)|broken_systemctl)::
 ?